src/renderers/WebGLRenderer.js

```diff
@@ -1147,6 +1151,61 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	}
 
+	this.renderOverride = function ( overrideMaterial, scene, camera, renderTarget, clearColor, clearAlpha ) {
+
+		var originalClearColor = this.getClearColor(), originalClearAlpha = this.getClearAlpha(), originalAutoClear = this.autoClear;
+
+		this.autoClear = false;
```

Comment:
That's what I mean. Better to leave the clearing logic to the user and better to make it explicit.